modules/form-field/react/lib/FormField.tsx

```diff
@@ -79,22 +80,19 @@ export default class FormField extends React.Component<FormFieldProps> {
     useFieldset: false,
   };
 
-  private renderChildren = (child: React.ReactChild): React.ReactNode => {
+  private renderChildren = (child: React.ReactElement): React.ReactNode => {
     if (React.isValidElement<any>(child)) {
       const props: GrowthBehavior &
         FormFieldErrorBehavior &
         React.HTMLAttributes<HTMLInputElement> = {
         ...child.props,
       };
 
-      if (this.props.grow && React.isValidElement<GrowthBehavior>(child)) {
```

Comment:
I don't think this check was doing anything since we are already wrapped in a React.isValidElement<any>(child) check